### PR TITLE
Detail crop portrait

### PR DESCRIPTION
**Pull request by mikeyrayvon**
hard crop portrait thumbs 16:9

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
my video sucker isn't working for the featured image. the image appears, and then disappears when i "Publish". and the image is never uploaded to media library...

so i cant test this

File: lib/thumbnail-sizes.php [hunk 1]
```diff
@@ -12,4 +12,5 @@
   add_image_size( 'product-thumb', 450, 450, false );
   add_image_size( 'item-l-4', 600, 800, false );
   add_image_size( 'gallery', 9999, 600, false );
+  add_image_size( 'portrait-16-9', 190, 337, true );
```
Comment (mixcas) [inline lib/thumbnail-sizes.php]: The dimensions are backwards

**Comment by mikeyrayvon** [comment]
oh lol

**Comment by mikeyrayvon** [comment]
190 is the max width tho @mixcas so it should be 106px height

**Pull request merged**